pkg/controllers/disruption/drift_test.go

```diff
@@ -151,56 +151,18 @@ var _ = Describe("Drift", func() {
 			ExpectMakeNodesAndNodeClaimsInitializedAndStateUpdated(ctx, env.Client, nodeStateController, nodeClaimStateController, nodes, nodeClaims)
 			ExpectSingletonReconciled(ctx, disruptionController)
 
-			metric, found := FindMetricWithLabelValues("karpenter_nodepools_allowed_disruptions", map[string]string{
+			metric, found := FindMetricWithLabelValues("karpenter_voluntary_disruption_decisions_total", map[string]string{
 				"nodepool": nodePool.Name,
+				"reason":   string(v1.DisruptionReasonDrifted),
 			})
-			Expect(found).To(BeTrue())
-			Expect(metric.GetGauge().GetValue()).To(BeNumerically("==", 10))
-
-			// Execute command, thus deleting all nodes
-			ExpectSingletonReconciled(ctx, queue)
-			Expect(len(ExpectNodeClaims(ctx, env.Client))).To(Equal(0))
-		})
-		It("should allow no empty nodes to be disrupted", func() {
```

Comment:
We're removing this because the drift budget no longer applies, right?

Reply:
Correct - although, with your other comments, doing a quick run through of all these tests instead of just the functionality makes sense.